### DISCUSSION TITLE
pull request

### DISCUSSION THREAD

**Discussion by KatsuHere** [community/community#144252] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

 How do I create a pull request?

**Comment by NyxianNetwork** [comment]
After pushing changes to a branch, go to the repository on GitHub, navigate to the "Pull requests" tab, and click "New pull request." Select the branches to merge from and into, and then submit.

**Reply by nyxiancode** [reply, marked as answer]
yap thanks
